pkg/daemon/update.go

```diff
@@ -2210,11 +2211,7 @@ func (dn *CoreOSDaemon) applyLayeredOSChanges(mcDiff machineConfigDiff, oldConfi
 	// Update OS
 	if mcDiff.osUpdate {
 		if err := dn.updateLayeredOS(newConfig); err != nil {
-			nodeName := ""
-			if dn.node != nil {
-				nodeName = dn.node.Name
-			}
-			MCDPivotErr.WithLabelValues(nodeName, newConfig.Spec.OSImageURL, err.Error()).SetToCurrentTime()
+			MCDPivotErr.WithLabelValues(newConfig.Spec.OSImageURL).Inc()
```

Comment:
Thanks for the review and good observation. 
Until 4.11, osImageURL was under our control as it would be populated with what is shipped through OCP release payload. With 4.12 onward where we moved to layering, we allow user to override OSImageURL field with their custom built OSImage. So, this could lead to unbound cardinality. I have updated commit with label removed, should be good now.